conu/apidefs/metadata.py

```diff
@@ -0,0 +1,48 @@
+class Metadata:
+    """
+    Common metadata for container and image
+    """
+
+    def __init__(self, name=None, id=None, ports=None, labels=None, command=None,
+                 creation_timestamp=None, env_variables=None):
+        self.name = name
+        self.id = id
+        self.ports = ports
```

Comment:
I would remove ports from this list since it has different meaning for container and images.